src/app/components/list/list.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { ListItem } from '../../modules/config/config.state';
+import { ListItemTile } from '../listItem/listItem.component';
+
+interface ListProps {
+    listItems: Array<ListItem>;
```

Comment:
List is too generic name, I'd suggest to use more specific name. E.g. staticContentItems if there would be a some base interface for staticContentItem which could be extended

Reply:
changed